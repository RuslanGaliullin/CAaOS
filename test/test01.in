dsdkjdakljsdkljd aksdj aksjlkjdk j
sdjdksdjl asd
asddkas jdklasdkjaksdj aasddsd
