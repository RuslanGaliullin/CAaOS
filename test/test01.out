Built array B:
2 4 6 8 10 12 1 3 5 7 9 11 

Calculation time = 1.1e-05
